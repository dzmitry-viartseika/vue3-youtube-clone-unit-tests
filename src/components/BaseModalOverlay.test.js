import { render } from '@testing-library/vue';
import BaseModalOverlay from './BaseModalOverlay';

it.only('renders with snapshot', () => {
    const { html } = render(BaseModalOverlay);
    const comment = 'A very very long comment';
    const videoObj = {
        id: Math.ceil(Math.random() * 100),
        title: 'Video title',
        publishedAt: new Date(),

    }
    expect(html()).toMatchSnapshot();
    expect(videoObj).toMatchSnapshot({
        id: expect.any(Number),
        publishedAt: expect.any(Date)
    });
    expect(comment).toMatchSnapshot();
});