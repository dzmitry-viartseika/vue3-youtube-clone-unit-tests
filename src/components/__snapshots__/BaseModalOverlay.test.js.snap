// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with snapshot 1`] = `"<div class=\\"fixed inset-0 bg-black bg-opacity-80\\" data-testid=\\"base-modal-overlay\\"></div>"`;

exports[`renders with snapshot 2`] = `
Object {
  "id": Any<Number>,
  "publishedAt": Any<Date>,
  "title": "Video title",
}
`;

exports[`renders with snapshot 3`] = `"A very very long comment"`;
